HACKING.md

```diff
@@ -42,6 +42,46 @@ to configure it for this repository.
 Tests can be run using `make test`, which will run all test forms. Specific types of
 tests can be run with other testing targets shown in `make help`.
 
+In addition to unit tests in `tests/unit`, which can be run with `make test-units`, a
```

Comment:
I think the first part of this sentence is incorrect, and in fact can be dropped. There is no `test-units` make target - you can run fast tests with `make test-fast`, slow tests with `make test-slow`, or both with `make test`. Since the previous paragraph already talks about `make test`, maybe this paragraph can get right into spread?

Reply:
Is this better? b0733f8